ui/src/style/layout/page.scss

```diff
@@ -51,3 +55,27 @@
   .container-fluid {padding: 8px !important;}
   .template-control--manage {display: none;}
 }
+
+/*
+  Dashboard Page
+  ------------------------------------------------------------------------------
+  Using a flex based layout so that the Template Variable Control Bar can
+  have any height without disrupting the layout
+*/
+.page.dashboard-page {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  flex-wrap: nowrap;
+
+  .page-header {
+    position: relative;
+  }
+  
+  .page-contents {
+    position: relative !important;
+    flex: 1 0 0;
+    height: 100% !important;
+    top: 0;
+  }
+}
```

Comment:
What's stopping us at this point from using flexbox for the whole dashboard `.page` layout (not including the sidebar)? Seems like you probably wouldn't need to use `position: relative` on the control bar + could avoid having to use precalculated height offsets?

Reply:
that thought crossed my mind as well, wasn't sure if making a broader change like that falls under the scope of this PR or not, but using flexbox would make everything a bit more flexible (no pun intended). What do you think?